File: java/ABTestExample/app/src/main/java/com/pubscale/sdkone/example/abtestexample/event_listener/InterstitialAdEventListener.java
package com.pubscale.sdkone.example.abtestexample.event_listener;

public interface InterstitialAdEventListener {
    void onAdLoading();
    void onAdLoaded();
    void onAdLoadFailed();
    void onAdShowFailed();
    void onAdOpened();
    void onAdClosed();
}

File: kotlin/ABTestExample/app/src/main/java/com/pubscale/sdkone/example/abtestexample/event_listener/InterstitialAdEventListener.kt
package com.pubscale.sdkone.example.abtestexample.event_listener

interface InterstitialAdEventListener {
    fun onAdLoading()
    fun onAdLoaded()
    fun onAdLoadFailed()
    fun onAdShowFailed()
    fun onAdOpened()
    fun onAdClosed()
}